test/test-util.js

```diff
@@ -10,9 +10,25 @@ describe('Unlimited Token', function () {
 
         const factory = await createTokenFactory();
         token = await unlimitedSupplyTokenContract(
-            factory.deployUnlimitedSupplyToken('Test Token', 'TST', await owner.getAddress()),
+            factory.deployUnlimitedSupplyToken('Test Token', 'TST', [], await owner.getAddress()),
         );
     });
+
+    it('Cannot mint when not in minter list', async () => {
+        expect(token.transfer(await receiver.getAddress(), parseEther('1000'))).to.be.revertedWith(
+            'ERC20: transfer amount exceeds balance',
+        );
+    });
+
+    it('Admin able to add new address to minter list', async () => {
+        const recept = await token.connect(owner).addMinter(await owner.getAddress());
+        expect(recept).to.not.be.undefined;
+    });
+
+    it('Non-admin not able to add new address to minter list', async () => {
```

Comment:
@GarfDev This test seems equal to the test on line 23. You are connecting the owner here, maybe thats whats wrong?